app/src/main/res/layout/activity_welcome.xml

```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!--  <RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
remove this commented code. Do think writing 36 lines of commented code to a community project is a good practice?